app.js

```diff
@@ -58,11 +59,10 @@ form.parse(req, function (err, fields, files) {
    } else {
     const filePath = JSON.parse(JSON.stringify(files));
     const workbook = filePath.nlcfile.path;
-    const data = fs.readFileSync(workbook, { encoding : 'utf8'});
-    const options = {headers : "text"};
+    csv({noheader: true,headers: ['text']}).fromFile(workbook).then((jsonObj)=>{
```

Comment:
I'm a little confused about `noheader` and `headers` both being defined this way - they seem to conflict?

> `noheader`: Indicating csv data has no header row and first row is data row. Default is false. See [header row](https://www.npmjs.com/package/csvtojson#header-row)

> `headers`: An array to specify the headers of CSV data. If `--noheader` is `false`, this value will override CSV header row. Default: null. Example: `["my field","name"]`. See [header row](https://www.npmjs.com/package/csvtojson#header-row)

Source: https://www.npmjs.com/package/csvtojson#api

This was also a little difficult to understand without the subsequent lines being correspondingly indented.

Reply:
@dolph I had followed the below info from the same source. https://www.npmjs.com/package/csvtojson#api

<img width="809" alt="screen shot 2018-11-20 at 8 20 54 pm" src="https://user-images.githubusercontent.com/11567379/48781338-d5a65c80-ed01-11e8-91b3-ddefc14ffd63.png">

Reply:
That's a really confusing API to me... but this example from the docs better explains what you're doing:

```javascript
// original source has no header row. use 'header1' 'header2' as its header row
csv({
    noheader: true,
    headers: ['header1','header2']
})
```

Which makes sense to me.